src/plugins/custom-save/Controller.ts

```diff
@@ -0,0 +1,22 @@
+import { desModderController } from "script";
+
+function updateView() {
+  desModderController.updateMenuView();
```

Comment:
Is this doing anything? Was under the impression this was to update DesModder menus/pillboxes, not Desmos's. Perhaps Calc.controller.updateViews()?